### PR TITLE
Added new advanced course for Next.js

### PR DESCRIPTION
**Pull request by Atharva-3000**
## What does this PR do?
Add resource(s)

## For resources
For Next.js

### Description
Added new video resource for React.js developers who want to learn Next.js.

### Why is this valuable (or not)?
This video is comprehensive, and covers all the necessary points that make Next.js the next big leap.

### How do we know it's really free?
It's on YouTube duhh ; )

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
when you've made changes, please add `(changes made)` to the title of your PR

**Comment by Atharva-3000** [comment]
On it !

**Comment by eshellman** [comment]
bad markdown syntax!
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -897,6 +897,7 @@
 * [Mastering Next.js](https://masteringnextjs.com)
 * [Next.js for Beginners - Full Course](https://www.youtube.com/watch?v=1WmNXEVia8I) - Kapehe (FreeCodeCamp)
 * [Next.js Tutorial for Beginners](https://youtube.com/playlist?list=PLC3y8-rFHvwgC9mj0qv972IO5DmD-H0ZH) - Codevolution
+* [Next.js Tutorial for React Devs](https://youtube.com/playlist?list=PL0Zuz27SZ-6Pk-QJIdGd1tGZEzy9RTgtj&si=aRI4WuATH55lYS1c) - Dave Gray
```
Comment (eshellman) [inline courses/free-courses-en.md:900]:
```suggestion
* [Next.js Tutorial for React Devs](https://youtube.com/playlist?list=PL0Zuz27SZ-6Pk-QJIdGd1tGZEzy9RTgtj) - Dave Gray
```
remove the tracking parameter

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -897,6 +897,7 @@
 * [Mastering Next.js](https://masteringnextjs.com)
 * [Next.js for Beginners - Full Course](https://www.youtube.com/watch?v=1WmNXEVia8I) - Kapehe (FreeCodeCamp)
 * [Next.js Tutorial for Beginners](https://youtube.com/playlist?list=PLC3y8-rFHvwgC9mj0qv972IO5DmD-H0ZH) - Codevolution
+* [Next.js Tutorial for React Devs](https://youtube.com/playlist?list=PL0Zuz27SZ-6Pk-QJIdGd1tGZEzy9RTgtj) - Dave Gray
```
Comment (eshellman) [inline courses/free-courses-en.md:900]: Don't invent titles! Use the title given by the resource.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Comment by Atharva-3000** [comment]
Thank you for assisting and keeping your cool with my silly mistakes.

**Pull request merged**